### ISSUE TITLE
Unable to use Chase.com with uMatrix and uBlock

### ISSUE THREAD

**Issue by Whoareyouagain** [gorhill/uMatrix#666]
I am having an weird issue with Chase. Everytime I log in, it always claimed I have Javascript disabled and Javascript is enabled by default. I disabled both extension and restart Chrome, I am able to get in my account information. With uMatrix and uBlock, it stop working and refused to let me go through. I whitelisted Chase in both extension, but it still will not let me through.

**Comment by gorhill** [comment]
Closing and marking as _invalid_, as per [CONTRIBUTING](https://github.com/gorhill/uMatrix/blob/master/CONTRIBUTING.md).

**Comment by Whoareyouagain** [comment]
I will add more details. It is uMatrix is blocking the javascript. I tested with uBlock without uMatrix enabled, it work just fine. It is uMatrix is causing the issue.

I log in my account with the user name and the password, it proceed to log. When it accept the user name and password, it redirect the page to JavaScript Disabled warning and asked me to turn it on. I tested it in Guest Mode (no addons) and it work just fine without uMatrix. All of the script is enabled in uMatrix, it still not allowing chase to function without the javascript. I have provide screenshot of the webpage and the uMatrix Script page. 

![chase javascript disabled](https://cloud.githubusercontent.com/assets/13504133/20472802/2bb95d0a-af71-11e6-8a58-fdd688add4d8.png)

![umatrix page](https://cloud.githubusercontent.com/assets/13504133/20472806/3345f286-af71-11e6-8399-2d56f10167b0.png)

**Comment by Rimmer7** [comment]
I find that oddball issues like this are usually fixed by disabling Strict HTTPS or more rarely User Agent/Referrer Spoofing for the scope. You can change those settings for chase.com by clicking the three blue dots to the left of the padlock.

**Comment by gorhill** [comment]
The logger should always being looked at, there is no word you did.

**Comment by jjohns71** [comment]
@Whoareyouagain I use chase as well and found after their recent site redesign, you need to enable some additional 3rd party access to get it to work properly. Try adding the following rules to your 'My rules' tab and see if the site works for you now.

`chase.com cdn.f9client.com frame allow`
`chase.com chasecdn.com script allow`
`chase.com chasecdn.com xhr allow`
`chase.com poc.clixmetrix.com frame allow`
`chase.com poc.clixmetrix.com script allow`
`chase.com secure.chasecdn.com cookie allow`
`chase.com use.typekit.net script allow`

**Comment by Whoareyouagain** [comment]
Sorry, I forgot to add the log. Here it is. I believe I have it fixed. the issue is I am denying script to chasecdn.com which cause it to claim the javascript is disabled. I disabled uMatrix and log in my account, then I reenable uMatrix to see which script is blocked. I discovered it was chasecdn.com. Odd enough, Chasecdn.com never show up in the list when I first load Chase.com. When I add chasecdn in approved section, chasecdn automatically show up in the list. That is why I never see chasecdn.com in the first place.


[Chase Log.txt](https://github.com/gorhill/uMatrix/files/604459/Chase.Log.txt)
